ClosedXML_Tests/Excel/CalcEngine/MathTrigTests.cs

```diff
@@ -217,6 +217,82 @@ public void Mod()
             Assert.AreEqual(0.7, actual, tolerance);
         }
 
+        /// <summary>
+        /// refers to Example 1 from the Excel documentation,
+        /// <see cref="https://support.office.com/en-us/article/SUMIF-function-169b8c99-c05c-4483-a712-1697a653039b?ui=en-US&rs=en-US&ad=US"/>
+        /// </summary>
+        /// <param name="expectedOutcome"></param>
+        /// <param name="formula"></param>
+        [TestCase(63000, "SUMIF(A1:A4,\">160000\", B1:B4)")]
+        [TestCase(900000, "SUMIF(A1:A4,\">160000\")")]
+        [TestCase(21000, "SUMIF(A1:A4, 300000, B1:B4)")]
+        [TestCase(28000, "SUMIF(A1:A4, \">\" &C1, B1:B4)")]
+        public void SumIf_ReturnsCorrectValues_ReferenceExample1FromMicrosoft(int expectedOutcome, string formula)
+        {
+            using(var wb = new XLWorkbook())
```

Comment:
Do a `Format Document` on all files before committing to ensure correct whitespace.

Reply:
did that now as well.